fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/values/simplification/Simplification.java

```diff
@@ -34,11 +34,11 @@
 import com.apple.foundationdb.record.query.plan.cascades.predicates.simplification.QueryPredicateComputationRuleCall;
 import com.apple.foundationdb.record.query.plan.cascades.predicates.simplification.QueryPredicateComputationRuleSet;
 import com.apple.foundationdb.record.query.plan.cascades.values.Value;
+import com.apple.foundationdb.record.util.pair.Pair;
```

Comment:
This does expose the `Pair` in a nominally public API. This should only be used internally, given that it's pretty deep in the planner, though the class isn't marked as such